src/components/Button/styles/styles.module.scss

```
@@ -0,0 +1,18 @@
+@import "../../../assests/variables.module.scss";
+
+.button {
+  padding: $space-md $space-lg;
+  border-width: 0;
+  border-radius: $space-md;
+  background-color: $tertiary-color;
+  color: $background-secondary-color;
+  font-size: $size-md;
+}
+
+.primary {
+  background-color: $primary-color;
+}
+
+.success {
+  background-color: $tertiary-color;
+}
```

Based on your code the `default` and `success` button have `$tertiary-color` and then the primary has `$primary-color`. If that's correct you can simplify it a bit. And use nesting feature of sass.
```suggestion
.button {
  padding: $space-md $space-lg;
  border-width: 0;
  border-radius: $space-md;
  background-color: $tertiary-color;
  color: $background-secondary-color;
  font-size: $size-md;
  
  &.primary {
    background-color: $primary-color;
  }
}
```
---
BTW, I suggest the other way around.
Keep the primary and default as same and add more. Something like:
```scss
.button {
  // general button styles
  // colors of default (primary)
  
  &.success {
    // colors of success
  }
  
  &.info {
    // colors of info
  }
}
```

---

thanksss